CHANGELOG.md

```diff
@@ -27,8 +27,8 @@ All notable changes to this project will be documented in this file. See [standa
 ### Features
 
 -   **addon-editor:** `Editor[new]` add custom tools support ([#842](https://github.com/TinkoffCreditSystems/taiga-ui/issues/842)) ([1422520](https://github.com/TinkoffCreditSystems/taiga-ui/commit/142252009a29e00ae4e92742abd12401109ef5a6))
--   **cdk:** new directive `TuiPan` ([#922](https://github.com/TinkoffCreditSystems/taiga-ui/issues/922)) ([d87996e](https://github.com/TinkoffCreditSystems/taiga-ui/commit/d87996e30516198e3b725b952c81cb3631fcb29a))
--   **cdk:** new directive `TuiSwipe` ([#914](https://github.com/TinkoffCreditSystems/taiga-ui/issues/914)) ([fb7581b](https://github.com/TinkoffCreditSystems/taiga-ui/commit/fb7581bee1058d35533b7d801cbabc092406eb76))
+-   **cdk:** new directive `Pan` ([#922](https://github.com/TinkoffCreditSystems/taiga-ui/issues/922)) ([d87996e](https://github.com/TinkoffCreditSystems/taiga-ui/commit/d87996e30516198e3b725b952c81cb3631fcb29a))
+-   **cdk:** new directive `Swipe` ([#914](https://github.com/TinkoffCreditSystems/taiga-ui/issues/914)) ([fb7581b](https://github.com/TinkoffCreditSystems/taiga-ui/commit/fb7581bee1058d35533b7d801cbabc092406eb76))
```

Comment:
why renamed? just interesting

Reply:
We do not use `Tui` prefix when naming components in text/docs, less noisy this way.